### ISSUE TITLE
Add a code of conduct (to discuss about)

### ISSUE THREAD

**Issue by elhmn** [osscameroon/roadmap#6]
Add a code of conduct that describes how to behave in the community.
The code of conduct that should also explains that adverts should only be posted in the telegram group by osscameroon active members, people that contributed with a PR or that supports us financially.

The code of conduct can be adapted from 
https://github.com/open-source-ideas/ideas/blob/master/CODE_OF_CONDUCT.md

This code of conduct was already extracted and adapted from https://www.contributor-covenant.org/version/1/4/code-of-conduct/

-----

This task will require us to edit the initial message presented to the people who join the telegram group.

Basically everyone joining the group should agree to the code of conduct

**Comment by monkeyK1n9** [comment]
Created a repo for that. [Check it out](https://github.com/osscameroon/who-we-are)

**Comment by tericcabrel** [comment]
Good job! We will iterate on that 👍🏾

**Comment by elhmn** [comment]
Could we move this repo you created to [this](https://github.com/osscameroon/.github/edit/master/profile/README.md) instead ?
